pkg/anypoint/client.go

```diff
@@ -53,6 +53,8 @@ type AnalyticsClient interface {
 	GetAnalyticsWindow() ([]AnalyticsEvent, error)
 	OnConfigChange(mulesoftConfig *config.MulesoftConfig)
 	GetClientApplication(appId string) (*Application, error)
+	GetLastRun() (string, string)
+	SaveLastRun(string)
```

Comment:
We probably don't need these on the AnalyticsClient. It can probably live on the Emitter and kept private.